File-Formats/Presentation/Working-with-sections.md

```diff
@@ -656,145 +480,102 @@ The following code example demonstrates how to clone the slide collection of a s
 {% tabs %}
 
 {% highlight c# tabtitle="C#" %}
-
 //Loads a PowerPoint presentation
 IPresentation pptxDoc = Presentation.Open("PPTXWithSections.PPTX");
-
 //Clones the slides in 3rd section
 ISlides slides = pptxDoc.Sections[2].Clone();
-
 //Creates a destination PowerPoint presentation instance. Existing presentations can also be used here.
 pptxDoc = Presentation.Create();
-
 //Iterates the cloned slides and adds the slides to the destination presentation
 foreach (ISlide slide in slides)
-
-pptxDoc.Slides.Add(slide);
-
+    pptxDoc.Slides.Add(slide);
 //Saves the PowerPoint presentation
 pptxDoc.Save("Sections.PPTX");
-
 {% endhighlight %}
 
 {% highlight vb.net tabtitle="VB.NET" %}
-
 'Loads a PowerPoint presentation
 Dim pptxDoc As IPresentation = Presentation.Open("PPTXWithSections.PPTX")
-
 'Clones the slides in 3rd section
 Dim slides As ISlides = pptxDoc.Sections(2).Clone()
-
 'Creates a destination PowerPoint presentation instance. Existing presentations can also be used here.
 pptxDoc = Presentation.Create()
-
 'Iterates the cloned slides and adds the slides to the destination presentation
-For Each slide As ISlide In slides
-
-pptxDoc.Slides.Add(slide)
-
+    For Each slide As ISlide In slides
```

Comment:
Please remove empty space before this line.

Reply:
@Suriya-Balamurugan -Done